qemu/tests/qemu_guest_agent.py

```diff
@@ -165,87 +163,16 @@ def _get_main_qga_version(self, session, vm):
     @error_context.context_aware
     def _get_latest_pkg(self):
```

Comment:
With this new change, there is no long the concept of the `latest`,  it is used to install the specified qga.

Reply:
they're moved into internal project, right? Did you check our internal project? pullrequest_514.

Reply:
No, i know that. I mean the concept is not `latest` now.  It is used to install the specified qga(/qemu-guest-agent*) with the given path, it depends on the user.

Reply:
so you mean this function should better change func_name, because it's no longer meaning original concept right? if I got your point correctly, what do you think about naming it as "get_qga_pkg", and add a comment line ‘#’ to explain that we obtain the latest qga package through our special tool.

Reply:
Yeap, that is what i mean.  And from my view, no need to comment that we use latest through other tools in upstream.

Reply:
okay, got it, and the name  "_get_qga_pkg" you think is okay, I'll change it